web/tests/dispatch.test.js

```diff
@@ -177,6 +178,15 @@ run_test("alert_words", ({override}) => {
     assert.ok(alert_words.has_alert_word("lunch"));
 });
 
+run_test("saved_replies", ({override}) => {
+    override(saved_replies, "add_saved_reply", noop);
+    override(saved_replies, "remove_saved_reply", noop);
+    const add_event = event_fixtures.saved_replies__add;
+    dispatch(add_event);
+    const remove_event = event_fixtures.saved_replies__remove;
+    dispatch(remove_event);
```

Comment:
```suggestion
    assert.equal(saved_replies.get_saved_replies().length, 0)
   
    const add_event = event_fixtures.saved_replies__add;
    dispatch(add_event);
    assert.equal(saved_replies.get_saved_replies().length, 1)
    
    const remove_event = event_fixtures.saved_replies__remove;
    dispatch(remove_event);
    assert.equal(saved_replies.get_saved_replies().length, 0)
```

Reply:
You need to test something after dispatching the event. Hence, the change.

Reply:
Updated to make it consistent with other tests:

```js
const add_event = event_fixtures.saved_replies__add;
    {
        const stub = make_stub();
        override(saved_replies, "add_saved_reply", stub.f);

        dispatch(add_event);
        console.log(stub);
        assert.equal(stub.num_calls, 1);
        assert_same(stub.get_args("event").event, add_event.saved_reply);
    }

    const remove_event = event_fixtures.saved_replies__remove;
    {
        const stub = make_stub();
        override(saved_replies, "remove_saved_reply", stub.f);

        dispatch(remove_event);
        assert.equal(stub.num_calls, 1);
        assert_same(stub.get_args("event").event, remove_event.saved_reply_id);
    }
```